### PR TITLE
feat(FRO-14): Quick Transcribe

### PR DESCRIPTION
**Pull request by adedotxn**

### CODE REVIEW

File: packages/web/src/components/Audio/SentAudio.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import SpeechIcon from '../../assets/speech_line.svg';
+import { PropTypes } from 'prop-types';
+import { useState, useEffect } from 'react';
```
Comment (OkoloJohnbosco) [inline packages/web/src/components/Audio/SentAudio.jsx:3]: Are there extra files in the Audio component folder?? The convention for naming components is 
Audio - index.jsx
           -index.module.css
           
Please readjust

File: packages/web/src/components/Loaders/LoadingDots.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styles from './LoadingDots.module.css';
```
Comment (OkoloJohnbosco) [inline packages/web/src/components/Loaders/LoadingDots.jsx:1]: change this from LoadingDots.jsx to index.jsx

File: packages/web/src/components/Loaders/LoadingDots.module.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.circle_one {
```
Comment (OkoloJohnbosco) [inline packages/web/src/components/Loaders/LoadingDots.module.css:1]: change this from LoadingDots.module.css to index.module.css

**Review by OkoloJohnbosco — CHANGES_REQUESTED** [review]
Please check all this. Thanks

File: packages/web/src/components/Audio/SentAudio.jsx (hunk 1 above)
Comment (adedotxn) [reply packages/web/src/components/Audio/SentAudio.jsx]: No, there are no extra files
alright

Comment (adedotxn) [reply packages/web/src/components/Audio/SentAudio.jsx]: I will create a new PR right now

**Comment by adedotxn** [comment]
@OkoloJohnbosco I have made the changes, I believe this PR will be automatically updated with the changes?

**Comment by adedotxn** [comment]
@OkoloJohnbosco  there was someone's file that had a lot of errors -- linting errors, using class instead of className and using mapping over an array that's not even in the file, I had to make updates to that file because it made me unable to make commits or even push after pulling

I have no idea how the file was able to be commited or merged but it was.

I hope these changes do not affect other files

**Comment by OkoloJohnbosco** [comment]
> @OkoloJohnbosco there was someone's file that had a lot of errors -- linting errors, using class instead of className and using mapping over an array that's not even in the file, I had to make updates to that file because it made me unable to make commits or even push after pulling
> 
> I have no idea how the file was able to be commited or merged but it was.
> 
> I hope these changes do not affect other files

Try pulling again

**Comment by onifs10** [comment]
@adedotxn pls can you add what page you implemented and link to the figma file to the description, Thanks

**Comment by adedotxn** [comment]
> @adedotxn pls can you add what page/feature you implemented and link to the figma file to the description, Thanks

@onifs10 I have added these to the description

**Comment by onifs10** [comment]
@adedotxn, i honestly cant see any thing implemented on this page . 

even the import icon is in the wrong place 

image

**Comment by adedotxn** [comment]
> @adedotxn, i honestly cant see any thing implemented on this page . 
> 
> even the import icon is in the wrong place 
> 
> image
> 

The design was just adjusted yesterday. When I built it, that was what was there and submission time was already too close for me to start rebuilding/adjusting ..and the icon works for importing audio

I can't do much because I need an API to work with for the conversation flow to make even sense

Also, if the sidebars is affecting it (in mobile view), that was not my component to build, I had to resist touching it

**Comment by onifs10** [comment]
> > @adedotxn, i honestly cant see any thing implemented on this page .
> > even the import icon is in the wrong place
> > image
> 
> The design was just adjusted yesterday. When I built it, that was what was there and submission time was already too close for me to start rebuilding/adjusting ..and the icon works for importing audio
> 
> I can't do much because I need an API to work with for the conversation flow to make even sense
> 
> Also, if the sidebars is affecting it (in mobile view), that was not my component to build, I had to resist touching it

alright

**Pull request merged**